.kinetic.travis.rosinstall

```diff
@@ -0,0 +1,12 @@
+- git:
```

Comment:
rename both to `.travis.rosinstall[.$ROS_DISTRO]`
($ROSINSTALL_FILENAME defaults to `.travis.rosinstall` for now)